app/views/catalog/_show_sidebar.html.erb

```diff
@@ -0,0 +1,5 @@
+<div id="sidebar-usage" class="sidebar">
+  <ul>
+    <li><span class="number"><%= rails_env? ? views : 'X' %> </span> <i class="bi bi-eye-fill"></i>views</li>
```

Comment:
Why not use `Rails.env.production?` here? I think it would clearer.

Reply:
+1